browser/resources/settings/brave_system_page/brave_system_page_index.ts

```diff
@@ -124,10 +124,7 @@ export class SettingsBraveSystemPageIndexElement extends
     // </if>
 
     views.push('memory')
-
-    if (this.showBatterySettings_) {
```

Comment:
Hmm I don't think we should be pushing this either, if the section is hidden

Reply:
Sorry which fix would you want? Doesn't work before does works now with this fix. Can do an alternate approach if you suggest one. Or can just close this of you prefer to have an error in console.

Reply:
Suggested one in the review comment - I think we should call `await flush()` before `queueMicrotask` in `hasBatteryChanged_`